Header header

uint32[] object_id
string[] object_name
string[] object_base_shape
uint32[] object_type
float64[] x1
float64[] y1
float64[] x2
float64[] y2
float32[] angle
bool[]    graspable
